### PR TITLE
increase resources in the chart with tolerations to schedule on more powerfull nodes

### PR DESCRIPTION
**Pull request by zorjak**

### CODE REVIEW

**Review by kudlaj — APPROVED** [review]

File: deployment/charts/universalnode/templates/statefulset.yaml [hunk 1]
```diff
@@ -58,11 +58,16 @@ spec:
...
+          effect: "NoSchedule"
   volumeClaimTemplates:
   - metadata:
       name: storage
     spec:
       accessModes: [ "ReadWriteOnce" ]
       resources:
         requests:
-          storage: 100Gi
+          storage: 2000Gi
```
Comment (luispdm) [inline deployment/charts/universalnode/templates/statefulset.yaml]: Put `2Ti`, so it is exactly 2 terabytes

File: deployment/charts/universalnode/templates/statefulset.yaml [hunk 2]
```diff
@@ -58,11 +58,16 @@ spec:
         volumeMounts:
         - name: storage
           mountPath: /app/.universalnode
+      tolerations:
+        - key: "pod"
+          operator: "Equal"
+          value: {{ .Values.tolerations }}
```
Comment (luispdm) [inline deployment/charts/universalnode/templates/statefulset.yaml]: You don't need to specify separate tolerations values, one is fine. If we merge your proposal we'd have to taint the node with 2 values. As per my understanding of the [kubernetes documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/), if the node is tainted with multiple key-values, for a pod to be deployed onto that node it must have all the key-values defined in its tolerations block

**Review by luispdm — CHANGES_REQUESTED** [review]

File: deployment/charts/universalnode/templates/statefulset.yaml (hunk 2 above)
Comment (zorjak) [reply deployment/charts/universalnode/templates/statefulset.yaml]: how much I understand this is going to be which node is tainted to what? maybe I am wrong. like one node (pod) is tainted to one and another with the second physical node?

File: deployment/charts/universalnode/templates/statefulset.yaml (hunk 1 above)
Comment (zorjak) [reply deployment/charts/universalnode/templates/statefulset.yaml]: ok

File: deployment/charts/universalnode/templates/statefulset.yaml (hunk 2 above)
Comment (zorjak) [reply deployment/charts/universalnode/templates/statefulset.yaml]: ok I think you are right. changed

**Review by luispdm — APPROVED** [review]

**Review by ccubu — APPROVED** [review]

**Pull request merged**